### PR TITLE
#3127 - Remove Request a Change POC Options

### PR DESCRIPTION
**Pull request by sh16011993**
### As a part of this PR, the following were completed:

- Remove the Income and Income Partner options from the formio Student Request Change form.
- Remove the related `json` files.

### Screenshot:

image

### CODE REVIEW

File: sources/packages/forms/src/form-definitions/studentrequestchange.json [hunk 1]
```diff
@@ -460,14 +460,6 @@
                 "value": "studentfinancialinformationappeal",
                 "label": "Financial Information"
               },
-              {
```
Comment (dheepak-aot) [inline sources/packages/forms/src/form-definitions/studentrequestchange.json:463]: Please also remove the variables in workflow load-assessment-consolidated-data. 

image

**Comment by dheepak-aot** [comment]
Looks good. Just one comment.

File: sources/packages/forms/src/form-definitions/studentrequestchange.json (hunk 1 above)
Comment (sh16011993) [reply sources/packages/forms/src/form-definitions/studentrequestchange.json:463]: Thanks for the explanation @dheepak-aot

**Review by dheepak-aot — APPROVED** [review]
Thanks for making the change. Looks good 👍

**Pull request merged**